explore/src/clue/scala/queries/common/ObsQueriesGQL.scala

```diff
@@ -28,21 +28,10 @@ object ObsQueriesGQL:
   @GraphQL
   trait SequenceOffsets extends GraphQLOperation[ObservationDB]:
     val document = s"""
-      fragment stepDataGN on GmosNorthStep {
+      fragment stepData on GmosNorthStep {
```

Comment:
This works for GmosSouth, too, even though it specifies it is for GmosNorthStep?

Reply:
Good catch. It does seem to work. But I'll revert, just in case.

Reply:
Maybe grackle is not enforcing that.